research/src/pages/contribute.mdx

```diff
@@ -0,0 +1,179 @@
+The design systems we've launched with we believe are representative of these criteria.
+We may not accept PRs contributing design systems that are lacking by these criteria.
+
+## Create Source JSON
+
+Create a JSON5 file in `/sources` for the design system, like `/antd.json5`.
+Add the `$schema` key pointing to our `design-system.schema.json5` schema and complete the required fields.
+
+```json5
+{
+  $schema: '../schemas/design-system.schema.json5',
+  name: 'Ant Design',
+  description: 'A design system with values of Nature and Determinacy for better user experience of enterprise applications.',
+  url: 'http://ant.design',
+  by: 'AFX',
+}
+```
+
+### Components
+
+Every UI component in the design system should be added to the components array:
+
+```json5
+{
+  $schema: '../schemas/design-system.schema.json5',
+  name: 'Ant Design',
+  description: 'A design system with values of Nature and Determinacy for better user experience of enterprise applications.',
+  url: 'http://ant.design',
+  by: 'AFX',
+  components: [{ name: 'Button' }],
+}
+```
+
+Do not document utility components.
+Only document components which render a user interface that can be experienced either visually, by keyboard, or narration.
+
+### Concepts
+
+Component concepts are terms used to described the appearance and behavior of a component.
+Things like `primary` for a `Button`.
+
+A term is a concept if the component can "be" that concept.
+For example, no one says a Button can be `size` but it can be `small`.
+Put another way, record "is" relationships which denote _qualities_, like "that button is small".
+Ignore "has a" relationships which denote _properties_, like "that button has a size".
+
+```json5
+  components: [
+    {
+      name: 'Button',
+      concepts: [
+        { name: 'primary', category: 'type' },
+      ],
+    },
+  ],
+```
+
+:bulb: Always use kebab-case for concept names.
+
+### Images
+
+An image of the concept should be included with each concept.
+This allows a visual aid to be shown when the concept might need to be clarified in the specifications.
+
+```json5
+  components: [
+    {
+      name: 'Button',
+      concepts: [
+        { name: 'primary', category: 'type', image: 'antd-button-primary.png' },
+      ],
+    },
+  ],
+```
+
+You can use a screenshot tool to capture these images directly from the design system.
+Here's an example image of the primary Button in Ant Design.
+
+<Image src="antd-button-primary.png" alt="Ant Design Primary Button" />
+
+If the image requires showing motion or interaction, considering using a gif tool.
+
+<Image src="sui-button-animated.gif" alt="Semantic UI Animated Button" />
+
+## Create Component Page
+
+Adding a component to Open UI is a research process.
+Because of this, two pages are created for each component; a research page and a proposal page.
+Create your component pages like so:
+
+```
+/src
+  /components
+    your-component.mdx           # proposal page
```

Comment:
I wonder if we want to make this explicit: `your-component.propsoal.mdx`

Reply:
Done.